pyinjective/ofac.json

```diff
@@ -1,48 +1,158 @@
 [
+  "0x01e2919679362dfbc9ee1644ba9c6da6d6245bb1",
```

Comment:
why do we remove `ini` files in favor of lists repo, but leave this `ofac.json` file in the sdk repo?